src/Widgets/WindowOverview.vala

```diff
@@ -85,7 +90,9 @@ public class Gala.WindowOverview : Clutter.Actor, ActivatableComponent {
 
                 if (window.window_type != Meta.WindowType.NORMAL &&
                     window.window_type != Meta.WindowType.DIALOG ||
-                    window.is_attached_dialog ()) {
+                    window.is_attached_dialog () ||
+                    (window_ids != null && !(window.get_id () in window_ids))
```

Comment:
I think you can remove `window_ids != null` check if you set initialize window_ids to be {} by default.

Reply:
I'm currently using the check here as an indicator whether to show all windows or only certain windows